wg-deployment/README.md

```diff
@@ -0,0 +1,39 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# Deployment Working Group
+
+WG Deployment covers developing, maintaining artifacts to deploy and operate Kubeflow.
+
+The [charter](charter.md) defines the scope and governance of the Deployment Working Group.
+
+## Meetings
+* Regular WG Meeting (Pacific AM): [Tuesdays at 08:00 PT (Pacific Time)]() (biweekly - every other Tuesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](http://bit.ly/kf-meeting-notes).
+* Regular WG Meeting (Pacific PM): [Wednesdays at 17:30 PT (Pacific Time)]() (biweekly - every other Tuesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](http://bit.ly/kf-meeting-notes).
+
+## Organizers
+
+* Jiaxin Shan (**[@Jeffwan](https://github.com/Jeffwan)**), AWS
+* Animesh Singh (**[@animeshsingh](https://github.com/animeshsingh)**), IBM
+* Adhita Selvaraj (**[@swiftdiaries](https://github.com/swiftdiaries)**), Cisco
+* Vaclav Pavlin (**[@vpavlin](https://github.com/vpavlin)**), Red Hat
+* Yannis Zarkadas (**[@yanniszark](https://github.com/yanniszark)**), Arrikto
```

Comment:
If @swiftdiaries is stepping out - maybe we see if he can suggest a replacement from Cisco, or clarify if Cisco has any stake in it?

Similarly I believe @yanniszark and Arritko are moving away from kfctl

Reply:
@mameshini is Agile stack interested in contributing and maintaining kfctl?

@vpavlin we haven't heard any revised information from RedHat? I believe you have. a stake and are interested in contributing and maintaining kfctl?

Reply:
@animeshsingh you are correct. Once the Manifests WG gets accepted, we (Arrikto) will move away from this WG.

Reply:
@animeshsingh Yes AgileStacks engineers will participate in kfctl enhancements and maintenance, with code contributions and testing of kfctl on AWS and on-prem.  I can be the point of contact for development priorities or PR reviews.

Reply:
Great. I will make some changes

Reply:
It is not my intention to dissuade anyone from being a lead; I also intend to let the leads self-organize. That said let me try to clearly communicate my expectation.

The expectation that I've tried to set in other WGs is that the leads are responsible for the success or failure of the projects.

This is intentionally & necessarily a higher bar then just contributing. My expectation is that leads have significant skin in the game and are willing and able to invest the resources needed to make their projects successful